web/src/main/webapp/WEB-INF/pages/candidates.jsp

```diff
@@ -10,11 +10,13 @@
 
 <atsy:secure_page>
     <jsp:attribute name="pageJs">
-     <c:url value="/resources/js/atsy-candidate.js" var="urlValue"/><script src="${urlValue}"></script>
-      <c:url value="/resources/js/atsy-deletable-event-config.js" var="urlValue" /><script src="${urlValue}"></script>
-              <c:url value="/resources/thirdparty/bootstrap/js/bootstrap.min.js" var="urlValue"/><script src="${urlValue}"></script>
-              <c:url value="/resources/thirdparty/bootbox/bootbox.js" var="urlValue"/><script src="${urlValue}"></script>
-              <c:url value="/resources/thirdparty/jquery/jquery.i18n.properties.js" var="urlValue"/><script src="${urlValue}"></script>
+        <c:url value="/resources/js/atsy-settings.js" var="urlValue"/><script src="${urlValue}"></script>
```

Comment:
atsy-settings.js is a dedicated javascript file for settings page. Not a good habit to reuse dedicated javascript files in different pages. The common javascript codes are collected in atsy.js file.